react/tic-tac-toe/src/app/screens/Main/screens/MatchHistory/index.js

```diff
@@ -23,11 +23,13 @@ class MatchHistory extends Component {
     const { hasError, isLoading, matchesHistory} = this.props;
     const matchesLines = hasError || !matchesHistory ? GET_ERROR : matchesHistory.map(this.renderLine);
     return (
-      <div className={styles.matchHistory}>
-      {MATCH_HISTORY_TITLE}
-        {isLoading ?
-          <Spinner className={styles.spinner}/>
-          : matchesLines}
+      <div className={styles.container}>
```

Comment:
You have a div with only one div like children. Is really necessary?

Reply:
That's because i wanted to center the match history, if i use only one div and i use align-items: center the items center but are not in the same line and i can't use margin auto because flex items take all the row. If i use display: inline-flex the items only use neccesary space but margin: auto is not working with inline items.

Reply:
I change it so that it is not centered, but I would like to know if you know any solution with a single div so that it is centered